D3 package is missing.

> ERROR in ./js/views/me.js
Module not found: Error: Cannot resolve module 'd3' in $HOME/Source/data-selfie/src/js/views
 @ ./js/views/me.js 10:9-22

--- a/package.json
+++ b/package.json
@@ -18,6 +18,7 @@
     "expose-loader": "^0.7.1",
     "jquery": "^3.1.1",
     "lodash": "^4.16.6",
+    "d3": "^4.5.0",
     "moment": "^2.17.0",
     "throttle-debounce": "^1.0.1",
     "webpack": "^1.13.3"

